bin/oneoff/nces_data/import_school_stats_by_years_2014_2015_ccd

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+CDO.log = Logger.new(STDOUT)
+
+AWS::S3.process_file('cdo-nces', '2014-2015/ccd/ccd_sch_029_1415_w_0216601a.txt') do |filename|
+  SchoolStatsByYear.transaction do
+    SchoolStatsByYear.merge_from_csv(filename) do |row|
+      {
+        school_id:          row['NCESSCH'].to_i.to_s,
+        school_year:        row['SURVYEAR'].presence || '2014-2015',
+        grades_offered_lo:  row['GSLO'],
+        grades_offered_hi:  row['GSHI'],
+        grade_pk_offered:   row['PKOFFERED'][0, 1] == 'Y',
```

Comment:
- `x[0,1]` (where `x` is a string) is equivalent to `x[0]`
- The substring isn't necessary unless you explicitly can't just match the whole string (e.g., you want to match both `Yes` and `Y`, but not `y` or `yes`). For any more complicated matching, a regex would probably be more customizable/readable.
- this matcher code could be refactored into a helper method to avoid copy/pasting, since it's reused many times in the script this would help readability